requirements.txt

```diff
@@ -1,5 +1,4 @@
 ratelim
-pyCrypto
-earthengine-api
+earthengine-api>=0.1.335
```

Comment:
Please change this to a specific version to avoid mismatch between EE versions for users, which will be harder to debug in case of bugs. For example, 1.20 would be good here.

Reply:
I think that it is not necessary due that those libs are distributed inside the final package (ran by the github action inside the docker), so users never install it directly. But, before the next release we need to update the "extlibs" and set up the latest versions of those libs